docs/excel/custom-functions-errors.md

```diff
@@ -58,12 +57,50 @@ let error = new CustomFunctions.Error(CustomFunctions.ErrorCode.invalidValue, "T
 throw error;
 ```
 
-## Use try-catch blocks
+### Output an array with an error
 
-In general, use `try`-`catch` blocks in your custom function to catch any potential errors that occur. If you do not handle exceptions in your code, they will be returned to Excel. By default, Excel returns `#VALUE!` for unhandled errors or exceptions.
+In addition to returning a single error, a custom function can output an array that includes an error. For example, a custom function could output the array `[1],[#NUM!],[3]`. The following code sample shows how to input three parameters into a custom function, replace one of the input parameters with a `#NUM!` error, and then return a 2-dimensional array with the results of processing each input parameter.
 
-In the following code sample, the custom function makes a fetch call to a REST service. It's possible that the call will fail, for example, if the REST service returns an error or the network goes down. If this happens, the custom function will return `#N/A` to indicate that the web call failed.
+```js
+/**
+* Returns the #NUM! error as part of a 2-dimensional array.
+* @customfunction
+* @param {number} first First parameter.
+* @param {number} second Second parameter.
+* @param {number} third Third parameter.
+* @returns {number[][]} Three results, as a 2-dimensional array.
+*/
+function returnInvalidNumberError(first, second, third) {
+  // Use the `CustomFunctions.Error` object to retrieve an invalid number error.
+  var error = new CustomFunctions.Error(
+    CustomFunctions.ErrorCode.invalidNumber, // Corresponds to the #NUM! error in the Excel UI.
+  );
+
+  // Enter logic that processes the first, second, and third input parameters.
+  // Imagine that the second calculation results in an invalid number error. 
+  var firstResult = first;
+  var secondResult =  error;
+  var thirdResult = third;
+
+  // Return the results of the first and third parameter calculations and a #NUM! error in place of the second result. 
+  return [[firstResult], [secondResult], [thirdResult]];
+}
+```
+
+### Errors as custom function inputs
+
+A custom function can evaluate even if the input range contains an error. For example, a custom function can take the range **A2:A7** as an input, even if **A6:A7** contains an error.
+
+To process inputs that contain errors, a custom function must have the JSON metadata property `allowErrorForDataTypeAny` set to `true`. See [Manually create JSON metadata for custom functions](custom-functions-json.md#metadata-reference) for more information.
```

Comment:
Will these be autogenerated for you, or do you always have to manually set this in the JSON metadata?

Reply:
This question is answered in the !IMPORTANT note on line 96. Does that note lack clarity? Do you think that note should be placed in a different location?

Reply:
I didn't notice the note, my bad. The note is good where it is. thanks!